cunumeric/array.py

```diff
@@ -3054,7 +3051,7 @@ def sum(
             UnaryRedCode.SUM,
```

Comment:
The `dtype` argument of `sum` is not passed through.

Reply:
dbb22229a813684621c0d0228f9171120e576b0b